public/app/core/components/TimePicker/TimePickerWithHistory.tsx

```diff
@@ -34,6 +35,12 @@ export const TimePickerWithHistory = (props: Props) => {
               onAppendToHistory(value, values, onSaveToStore);
               props.onChange(value);
             }}
+            onPasteError={(clipboardContent: string) =>
```

Comment:
this is _hyper_ specific to the error you're trying to catch. what happens if we want to catch other errors? do we add a new independent prop for each one? 😅 

could we instead make it a more generic `onError` callback, then you pass these messages to the callback from `TimeRangeContent`